File: DSA_basics/strings/q10/solution.cpp
#include <iostream>
#include <cctype>
using namespace std;

char caseChange(char c) {
    if (islower(c)) {
        return toupper(c);
    }
    else if (isupper(c)) {
        return tolower(c);
    }

    return c;
}

int main() {
    string s = "hELLO wORLD";

    for (int i = 0; i < s.size(); i++) {
        s[i] = caseChange(s[i]);
    }

    cout << s << endl;

    return 0;
}

File: DSA_basics/strings/q10/solution.py
def case_change(c):
    if c.islower():
        return c.upper()
    elif c.isupper():
        return c.lower()
    return c

s = "hELLO wORLD"

result = ""
for ch in s:
    result += case_change(ch)

print(result)